assets/router/service-internal.yaml

```diff
@@ -5,12 +5,8 @@ apiVersion: v1
 metadata:
   # Name and Annotations are set at runtime.
   namespace: openshift-ingress
```

Comment:
Specifying `namespace` in the asset is redundant now that `desiredInternalIngressControllerService` sets it.

Reply:
Fixed